modules/data.atmosphere/DESCRIPTION

```diff
@@ -61,10 +61,11 @@ Imports:
     zoo
 Suggests:
     doParallel,
-    foreach,
+    doSNOW,
     furrr,
     future,
     getPass,
+    ggplot2,
```

Comment:
In principle I support moving ggplot to Suggests, but have you checked where it's currently used and whether we need to make any code changes to have it be actually optional?